base/path_service_unittest.cc

```diff
@@ -136,10 +139,22 @@ TEST_F(PathServiceTest, Get) {
   constexpr std::array<BasePathKey, 0> kUnsupportedKeys = {};
 #endif  // BUILDFLAG(IS_ANDROID)
   for (int key = PATH_START + 1; key < PATH_END; ++key) {
+#if defined(STARBOARD)
```

Comment:
For Starboard do the unsupported keys not return invalid paths?

Reply:
There is a NOTREACHED() check which will fail the test immediately.